### DISCUSSION TITLE
Expose Laravel project without spesific port

### DISCUSSION THREAD

**Discussion by yuxxeun** [laravel/framework#49882] [Q&A]
It is possible to make my Laravel based project accessed without spesific port like `:8000` for example, which is laravel deault port, right?
I'm consern about this because i've build some laravel app for my company, and then all of them will deployed to my physical server company because i want to make them just only can be accessed on my company network via ip addresse i.e `10.10.10.10/my-laravel-project` (like `php` project on `htdocs` directory or `CodeIgniter` for example).

So far, what was i did is, run artisan command: `php artisan serve --host 10.10.10.10 --port 8000` and then it can be accessed on `http:10.10.10.10:8000` and i want to change it became `http://10.10.10.10/my-laravel-project`, it is possible?

**Comment by sav4n** [comment]
**Yes** 
You can try this 
`php artisan serve --host 10.10.10.10 --port 80`
if the port is taken with another process. 
I recommend you install nginx or apache to host this on specific port.
